### PR TITLE
docs: Modify the introduction of the homepage ReadMe

### PR DESCRIPTION
**Pull request by zhenlei520**
### Docs
* HomePage Readme: Modify the introduction of the homepage ReadMe

### Chores
* Masa.Utils.DotNet: Library renamed to Masa.Utils.Extensions.DotNet 
* Masa.Utils.Enums: Library renamed to Masa.Utils.Extensions.Enums

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,82 @@
...
+
+Thanks to all the friends who have contributed to this project.
+
+<a href="https://github.com/masastack/MASA.Utils/graphs/contributors">
+    <img src="https://contrib.rocks/image?repo=masastack/MASA.Utils" />
+</a>
+
+
+
+## code of conduct
```
Comment (doddgu) [inline README.md:72]: Code

File: README.zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+中 | [EN](README.md)
+
+# MASA.Utils
+
+MASA.Utils是MASA的工具库、这些工具被[MASA Stack](https://github.com/masastack)和[MASA Labs](https://github.com/masalabs)等项目使用。
```
Comment (doddgu) [inline README.zh-CN.md:5]: 、 -> ,

File: README.zh-CN.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+* [发行说明](https://github.com/masastack/MASA.Utils/releases)
+* [最新路线图](https://github.com/masastack/MASA.Utils/issues/40)
+
+
+
+## 特性
+* Caching
+  * Memory: 内存缓存
+  * Redis
+  * DistributedMemory: 分布式缓存
```
Comment (doddgu) [inline README.zh-CN.md:19]: 分布式内存缓存

File: README.md [hunk 4]
```diff
@@ -1 +1,82 @@
...
+* [Release Notes](https://github.com/masastack/MASA.Utils/releases)
+* [Latest Roadmap](https://github.com/masastack/MASA.Utils/issues/40)
+
+
+
+## Features
+* Caching
+  * Memory: memory cache
+  * Redis
+  * DistributedMemory: Distributed cache
```
Comment (doddgu) [inline README.md:19]: Distributed memory cache

**Review by doddgu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zhenlei520) [reply README.md]: processed

File: README.zh-CN.md (hunk 2 above)
Comment (zhenlei520) [reply README.zh-CN.md]: processed

File: README.zh-CN.md (hunk 3 above)
Comment (zhenlei520) [reply README.zh-CN.md]: processed

File: README.md (hunk 4 above)
Comment (zhenlei520) [reply README.md]: processed

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**